lib/service_manager.dart

```diff
@@ -102,6 +138,17 @@ class ServiceConnectionManager {
     _stateController.add(null);
     _connectionClosedController.add(null);
   }
+
+  // TODO(kenzie): add hot restart method, register method in flutter_tools.
```

Comment:
@aam would be a good one to coordinate with here

Reply:
Btw @aam helpe me figure out how to call this service extension from the devtools web app.